tools/build_chain.sh

```diff
@@ -408,8 +426,12 @@ gen_node_cert_with_extensions_gm() {
 
     $TASSL_CMD genpkey -paramfile "$capath/gmsm2.param" -out "$certpath/gm${type}.key" 2> /dev/null
     $TASSL_CMD req -new -subj "/CN=$name/O=fisco-bcos/OU=${type}" -key "$certpath/gm${type}.key" -config "$capath/gmcert.cnf" -out "$certpath/gm${type}.csr" 2> /dev/null
-    $TASSL_CMD x509 -req -CA "$capath/gmagency.crt" -CAkey "$capath/gmagency.key" -days 3650 -CAcreateserial -in "$certpath/gm${type}.csr" -out "$certpath/gm${type}.crt" -extfile "$capath/gmcert.cnf" -extensions "$extensions" 2> /dev/null
-    cat "${capath}/gmagency.crt" >> "$certpath/gm${type}.crt"
+    if [ -n "${no_agency}" ];then
+        echo "not use $(basename $capath) to sign $(basename $certpath) ${type}" >>"${logfile}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5344398)